packages/aws-client/tests/test-SQS.js

```diff
@@ -100,3 +101,23 @@ test('isSQSRecordLike filters correctly for sqs record shape', (t) => {
   body *should* be a string form json object,
   but strictly checking is not compatible with the current use-case*/
 });
+
+test('sendSQSMessage truncates oversized messages safely', async (t) => {
+  const queueName = randomString();
+  const queueUrl = await createQueue(queueName);
+  const maxLength = 262144;
+  const overflowMessage = '0'.repeat(maxLength + 2);
+  await sendSQSMessage(queueUrl, overflowMessage);
+
+  let recievedMessage = await receiveSQSMessages(queueUrl);
+  let messageBody = recievedMessage[0].Body;
+  t.true(messageBody.endsWith('...TruncatedForLength'));
```

Comment:
Do you normally add negative test cases? Like if the message is under the max size, then verify the '...TruncatedForLength' message does not appear. In the event it adds it to all the messages regardless of size.

Reply:
that's a good point, I'll add it

Reply:
Maybe at the end add:

Reply:
sure, we're checking implicitly just by checking that "it's unchanged", but verbosity==good in testing